teuthology/suite/test/test_run_.py

```diff
@@ -130,6 +130,45 @@ def test_sha1_nonexistent(
         with pytest.raises(ScheduleFailError):
             self.klass(self.args)
 
+    @patch('teuthology.suite.run.util.fetch_repos')
+    @patch('teuthology.suite.run.util.git_branch_exists')
+    @patch('teuthology.suite.run.util.package_version_for_hash')
+    @patch('teuthology.suite.run.util.git_ls_remote')
+    def test_regression(
+        self,
+        m_git_ls_remote,
+        m_package_version_for_hash,
+        m_git_branch_exists,
+        m_fetch_repos,
+    ):
+        config.gitbuilder_host = 'example.com'
+        m_package_version_for_hash.return_value = 'ceph_hash'
+        m_git_branch_exists.return_value = True
+        m_git_ls_remote.return_value = "suite_branch"
+        self.args_dict = {
+            'base_yaml_paths': [],
+            'ceph_branch': 'master',
+            'machine_type': 'smithi',
+            'kernel_flavor': 'basic',
+            'kernel_branch': 'testing',
+            'suite': 'krbd',
+        }
+        self.args = YamlConfig.from_dict(self.args_dict)
+        with patch.multiple(
+            'teuthology.suite.util.GitbuilderProject',
+            _get_package_sha1=DEFAULT,
+        ) as m:
+            assert m != dict()
```

Comment:
Yeah. It's left over from development and I forgot to remove it. I can remove it if you want?